### ISSUE TITLE
ipfs.refs.local(); Exception

### ISSUE THREAD

**Issue by cheershendtco** [ipfs/http-api-spec#121]
i got exception at this line ipfs.refs.local();
how can i fix.
thanks.

`public class MyTest {

```
public static void main(String[] args) throws IOException {
    // TODO Auto-generated method stub

    APITests tt= new APITests();
    //tt.base58Test();
    IPFS ipfs = new IPFS("/ip4/127.0.0.1/tcp/5001");
    //IPFS ipfs = new IPFS(new MultiAddress("/ip4/127.0.0.1/tcp/5001"));
    ipfs.refs.local();//MyTest.java:15
```

`

`
Exception in thread "main" java.lang.IllegalStateException: json object at at 0  'QmNLtXUjLqKpoZqrkj16793SZGjzMu9xd84iWuPFDdQbnc
QmNNAgD3SydsjETJb6U9wdccgFGW45jHf5dnsr7R7Z4Fp7
QmNNTJYtCRd8xnP8EmeZpYcQ6EDmV7o89sDHem5HSLCKuT
    at org.ipfs.api.JSONParser.parse(JSONParser.java:254)
    at org.ipfs.api.JSONParser.parseStream(JSONParser.java:279)
    at org.ipfs.api.IPFS$Refs.local(IPFS.java:122)
    at org.ipfs.api.MyTest.main(MyTest.java:15)

`

**Comment by cheershendtco** [comment]
problem solved after i use v0.4.3-rc3

**Comment by RichardLitt** [comment]
Glad this was fixed. It looks like this question may should be in github.com/ipfs/support.
